django/cantusdb_project/main_app/management/commands/sync_sequences.py

```diff
@@ -76,9 +76,9 @@ def get_new_sequence(seq_id):
         folio = None
 
     try:
-        sequence_number = json_response["field_sequence_text"]["und"][0]["value"]
+        s_sequence = json_response["field_sequence_text"]["und"][0]["value"]
```

Comment:
`"sequence_number"` was incorrect, this substitution corrects the error (also line 81)